client/src/app/services/plan-my-services/plan-my-services.component.html

```diff
@@ -40,12 +40,28 @@ <h2>Ajouter mes services ...</h2>
                 <div>
                     <mat-list *ngIf="dayServices">
                         <mat-list-item *ngFor="let service of dayServices" fxLayoutAlign="start center">
-                            <p mat-line>
-                                <button type="button" mat-icon-button (click)="addService(service)">
-                                    <mat-icon>add</mat-icon>
-                                </button>
+
+                            <p mat-line *ngIf="available(service); else elseBlock">
+
+                                
+                                    <button type="button" mat-icon-button (click)="addService(service)">
```

Comment:
Why don't you use `ngIf` in the button tag (balise) ? As a result, you will write ` De {{service.startAt | date:'H'}}h à {{service.endAt | date:'H'}}h` only once.